src/components/connection-modal/dots.jsx

```diff
@@ -5,6 +5,20 @@ import classNames from 'classnames';
 import Box from '../box/box.jsx';
 import styles from './connection-modal.css';
 
+const Dot = props => (
+    <div
+        className={classNames(
+            styles.dot,
+            {
+                [styles.inactiveStepDot]: props.type === 'inactive',
+                [styles.activeStepDot]: props.type === 'active',
+                [styles.successDot]: props.type === 'success',
+                [styles.errorDot]: props.type === 'error'
```

Comment:
not essential, but could be nice to type `props.type` as a string literal union or enum so typos are easier to catch